js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeLineageGraph.tsx

```diff
@@ -149,30 +142,41 @@ export const AssetNodeLineageGraph = ({
                     style={{overflow: 'visible'}}
                     onMouseEnter={() => setHighlighted([id])}
                     onMouseLeave={() => setHighlighted(null)}
-                    onClick={() => onClickAsset({path})}
                     onDoubleClick={(e) => {
                       viewportEl.current?.zoomToSVGBox(bounds, true, 1.2);
                       e.stopPropagation();
                     }}
                   >
-                    {!graphNode ? (
-                      <AssetNodeLink assetKey={{path}} />
-                    ) : scale < MINIMAL_SCALE ? (
-                      <AssetNodeContextMenuWrapper {...contextMenuProps}>
-                        <AssetNodeMinimal
-                          definition={graphNode.definition}
-                          selected={graphNode.id === assetGraphId}
-                          height={bounds.height}
-                        />
-                      </AssetNodeContextMenuWrapper>
-                    ) : (
-                      <AssetNodeContextMenuWrapper {...contextMenuProps}>
-                        <AssetNode
-                          definition={graphNode.definition}
-                          selected={graphNode.id === assetGraphId}
-                        />
-                      </AssetNodeContextMenuWrapper>
-                    )}
+                    <Link
+                      to={assetDetailsPathForKey(
+                        {path},
+                        {
+                          ...params,
+                          lineageScope: 'neighbors',
+                          lineageDepth: 1,
+                        },
+                      )}
+                      style={{textDecoration: 'none'}}
+                    >
+                      {!graphNode ? (
+                        <AssetNodeLink assetKey={{path}} />
+                      ) : scale < MINIMAL_SCALE ? (
+                        <AssetNodeContextMenuWrapper {...contextMenuProps}>
+                          <AssetNodeMinimal
+                            definition={graphNode.definition}
+                            selected={graphNode.id === assetGraphId}
+                            height={bounds.height}
+                          />
+                        </AssetNodeContextMenuWrapper>
+                      ) : (
+                        <AssetNodeContextMenuWrapper {...contextMenuProps}>
+                          <AssetNode
+                            definition={graphNode.definition}
+                            selected={graphNode.id === assetGraphId}
+                          />
```

Comment:
I think we hadn't done this previously because this AssetNode contains links, and the DOM doesn't allow `<a>` tags within `<a>` tags :( At least, it was making React spew logs about invalid nesting.

Reply:
hmm interesting, I guess I could read e.metaKey and keep it in JS